lib/modules/apostrophe-login/views/passwordResetRequest.html

```diff
@@ -26,7 +26,7 @@
               <input type="submit" class="apos-button apos-button--major" value="{% block buttonLabel %}{{ __('Request Password Reset') }}{% endblock %}" />
             </div>
             <div class="apos-login-reset-cancel">
-              <a href="{{ apos.prefix }}/login">{{ __('Cancel') }}</a>
+              <a href="{{ apos.prefix }}{{data.loginUrl}}">{{ __('Cancel') }}</a>
```

Comment:
Where is this getting into the template data?

Reply:
https://github.com/apostrophecms/apostrophe/pull/2124/commits/32337ca76f483931dd45935611858115cff82fa6#diff-297b2881cc29c9a42671c4bc64f95986R454 this?